pageobjects/ForgotPasswordPage.js

```diff
@@ -0,0 +1,79 @@
+const { getToken } = require('../utilities/utilities');
```

Comment:
Curious why there are now 2 identical page objects - `forgotPasswordPage` and `ForgotPasswordPage`?

I'm guessing that this one was accidentally committed.

Reply:
oh dear....will fix that